api/definitions.yaml

```diff
@@ -119,11 +119,17 @@ PaymentNoticeNumber:
   pattern: "^[0123][0-9]{17}$"
 PaymentAmount:
   description: |-
-    Amount of payment in euro cent. Even if pagoPA accepts until 999999999999 euro cents,
-    we choose to limit the amount up to 100k cents (1k euros).
+    Amount of payment in euro cent. PagoPA accepts until 999999999 euro cents.
   type: integer
   minimum: 1
-  maximum: 100000
+  maximum: 9999999999
+MaxAllowedPaymentAmount:
```

Comment:
it may make sense to have the unit of measure as part of the name to avoid any possible misunderstanding - you know, people don't always read the docs

Reply:
we can but it will be a breaking change for some other projects that refers to these types linking to the swagger specs, so in case, we must alert all the other teams